jetbrains-core/src/software/aws/toolkits/jetbrains/ui/feedback/FeedbackDialog.kt

```diff
@@ -32,19 +38,31 @@ class FeedbackDialog(project: Project) : DialogWrapper(project) {
 
             val sentiment = panel.sentiment ?: throw IllegalStateException("sentiment was null after validation")
             val comment = panel.comment ?: throw IllegalStateException("comment was null after validation")
-            ApplicationManager.getApplication().executeOnPooledThread {
+            launch(coroutineContext) {
```

Comment:
I think the `coroutineContext` is redundant. That argument is for *additional* context.

> The coroutine context is inherited from a CoroutineScope. Additional context elements can be specified with context argument.

https://kotlin.github.io/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines/launch.html